stdlib/ssl.pyi

```diff
@@ -145,6 +145,7 @@ if sys.version_info >= (3, 6):
         OP_NO_TICKET: int
         if sys.version_info >= (3, 7):
             OP_NO_RENEGOTIATION: int
+        if sys.version_info >= (3, 6):
```

Comment:
There is already a `sys.version_info >= (3, 6)` guard around the class, so you could probably just remove this guard and move the line it to line 141, where it would fit better.

Same for the other change

Reply:
Quite so, thank you.